doc/content/source/actions/PolarPhaseFieldAction.md

```diff
@@ -0,0 +1,10 @@
+# PolarPhaseFieldAction
+
+!syntax description /Magpie/PolarPhaseField/PolarPhaseFieldAction
+
+This action simplifies the setup of the [polar phase field model](/PolarPhaseField/index.md)
```

Comment:
Should this say what `kernel`s this action sets up

Reply:
I'd also add a practical application for the use of this kernel/action

Reply:
There are verification examples and a test.

Reply:
I guess I was referring to elaborating on potentially practical applications. The examples aren't documented, so it doesn't help much.

Reply:
good point

Reply:
I added an explanation for the physical meaning of the model. For more details I would refer the reader to the original publication, which is cited.